games-roguelike/moria/moria-5.7.10.ebuild

```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit cmake-utils
+
+DESCRIPTION="Rogue-like D&D curses game similar to nethack (BUT BETTER)"
+HOMEPAGE="https://umoria.org/"
+SRC_URI="https://github.com/dungeons-of-moria/umoria/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~x86-fbsd"
+IUSE=""
+
+RDEPEND=">=sys-libs/ncurses-6:0"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+S="${WORKDIR}/umoria-${PV}"
+BUILD_DIR="${S}"
+
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_INSTALL_PREFIX="${EPREFIX}/usr/share"
+	)
+	cmake-utils_src_configure
+}
+
+src_install() {
+	cmake-utils_src_install
+	dodoc README.md docs/*
+
+	insinto /usr/share/umoria/data
+	doins data/*.txt
+
+	insinto /usr/share/umoria
+	doins LICENSE data/scores.dat
```

Comment:
we do not install LICENSE files

Reply:
I know, I had to. The binary tries to open it, see: https://github.com/gentoo/gentoo/pull/10899#discussion_r251069710